README.md

```diff
@@ -24,12 +23,12 @@ using MultiPrecisionR2
 using ADNLPModels
 using IntervalArithmetic
```

Comment:
Yes, I'm actually thinking about setting relative error model with classic evaluation by default instead of Interval. Interval by default is not very user-friendly (typically the `setrounding(Interval, :accurate)` has to be added). What do you think ?
I would simply use `sqrt(eps(T))` as default relative error for obj and grad evaluations.